lib/widgets/nine_tile_box.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/widgets.dart' as widgets;
+import 'package:meta/meta.dart';
+import '../sprite.dart';
+import 'dart:ui';
+
+class _Painter extends widgets.CustomPainter {
+  final Image image;
+  final double tileSize;
+  final double destTileSize;
+
+  _Painter(
+      {@required this.image,
+      @required this.tileSize,
+      @required this.destTileSize});
+
+  @override
+  void paint(Canvas canvas, Size size) {
+    final topLeftCorner =
+        Sprite.fromImage(image, x: 0, y: 0, width: tileSize, height: tileSize);
```

Comment:
how about a getImageTime method or something like that that takes two ints x and y, multiply them by tileSize and do the rest and returns the Sprite?

Reply:
Fair enought, done!